pkg/ReadMe.md

```diff
@@ -0,0 +1,84 @@
+# How to make a release
+
+## Versioning
+
+Update libyaml version in:
+* announcement.msg
+* Changes
+* CMakeLists.txt
+  * `YAML_VERSION_MAJOR`, `YAML_VERSION_MINOR`, `YAML_VERSION_PATCH`
+* .appveyor.yml
+* configure.ac
+  * `YAML_MAJOR`, `YAML_MINOR`, `YAML_PATCH`, `YAML_RELEASE`, `YAML_CURRENT`, `YAML_REVISION`
+
+Commit and push everything to release/0.x.y
+
+## Test and Create Release archives
+
+### GitHub Actions Automation
+
+The github workflow
+
+    .github/workflows/dist.yaml
+
+will do this automatically for you.
+
+#### .github/workflows/dist.yaml
+
+Will create release archives (tar.gz and zip)
+
+### Manually
+
+Make sure you have a clean git repository (no changed files).
+The following process will clone your current git directory.
+
+This will need the docker image `yamlio/libyaml-dev`.
+You can either pull it, or create it yourself:
+
+    make docker-build
+
+### Create dist archives
+
+Run:
+
+    make docker-dist
+
+It will run `make dist` in the container to create a tarball written to
```

Comment:
One line please.

Reply:
Then it would be over 80 characters long. I don't want that.

People using vim would have to manually fix this line after using `gq`. Why do you want exactly this line to be longer than 80 characters?

Reply:
I understood the point of one sentence per line, but if the sentence is longer than 80 chars, there should be no line break at all in a sentence? That seems a bit weird.

Reply:
I hadn't thought of mixing one sentence per line with line breaking. It's an interesting idea although 80 columns might be too aggressive.

Leave this as it is for now.